lib/features/splash/presentation/cubit/splash_state.dart

```diff
@@ -0,0 +1,14 @@
+part of 'splash_cubit.dart';
+
+abstract class UserState {
+  const UserState();
+}
+
+class Loading extends UserState {
+  const Loading();
+}
+
+class Loaded extends UserState {
+  const Loaded(this.user);
+  final bool user;
+}
```

Comment:
look for how to create a state with freezed is quite similar on what you have here but give us more tools to work

Reply:
Done